stumpy/stumpi.py

```diff
@@ -124,10 +124,25 @@ def __init__(self, T, m, egress=True, normalize=True, p=2.0):
 
         self._T, self._M_T, self._Σ_T = core.preprocess(self._T, self._m)
         # Retrieve the left matrix profile values
-        for i, j in enumerate(self._left_I):
-            if j >= 0:
-                D = core.mass(self._T[i : i + self._m], self._T[j : j + self._m])
-                self._left_P[i] = D[0]
+
+        # if left neighbor of `S[i] = T[i, i+m]` is actually its matrix profile index
```

Comment:
Maybe change to:
"Since each matrix profile value is the minimum between the left and right matrix profile values, then we can save time by only recomputing the left matrix profile value only when the matrix profile value is equal to the right matrix profile value."

Reply:
Sure. will revise it.

Reply:
Are you okay with this?
"Since each matrix profile value is the minimum between the left and right matrix profile values, we can save time by re-computing the left matrix profile value only when the matrix profile index is equal to right matrix profile index."

(Since we do not have the left/right matrix profile value but just their indices, I thought it would be better to avoid it in the comment)